behandlingslager/domene/src/main/java/no/nav/foreldrepenger/behandlingslager/behandling/aksjonspunkt/AksjonspunktDefinisjon.java

```diff
@@ -266,7 +266,7 @@ public enum AksjonspunktDefinisjon implements Kodeverdi {
         ENTRINN, EnumSet.of(FP)),
     VURDER_SVP_TILRETTELEGGING(
             AksjonspunktKodeDefinisjon.VURDER_SVP_TILRETTELEGGING_KODE, AksjonspunktType.MANUELL, "Vurder tilrettelegging svangerskapspenger",
-            BehandlingStegType.KONTROLLER_FAKTA, VurderingspunktType.UT, UTEN_VILKÅR, SkjermlenkeType.PUNKT_FOR_SVP_INNGANG, ENTRINN, EnumSet.of(ES, FP, SVP)),
+            BehandlingStegType.VURDER_SVANGERSKAPSPENGERVILKÅR, VurderingspunktType.INN, UTEN_VILKÅR, SkjermlenkeType.PUNKT_FOR_SVP_INNGANG, ENTRINN, EnumSet.of(ES, FP, SVP)),
```

Comment:
Hvorfor endrer man Behandling steg type? Jeg vet ikke helt hvordan prosessen med fakta og prosess henger sammen, men dette er vel vurdering av fakta, vilkåret vurderes i et senere aksjonspunkt?

Reply:
Dette er eneste aksjonspunktet som er satt opp til å løses ved KOFAK.UT (før du går ut av steget). Alle andre inngangsaksjonspunkt er satt opp til å løses før inngangen til steget som skal vurdere det aktuelle vilkåret
Det er derfor et unntak som vi vil fjerne (for å få startpunktlogikken til å bli mer strømlinjeformet og lik FP)